tests/foreman/cli/test_activationkey.py

```diff
@@ -319,10 +320,20 @@ def test_negative_create_with_usage_limit_with_not_integers(self):
             for value in invalid_values_list()
             if not value.isdigit()
         ]
+        content_to_validate = []
         invalid_values.append(0.5)
+        if invalid_values == "\t":
+            content_to_validate.append(
+                u'Max hosts cannot be nil')
+        if invalid_values == "0.5":
```

Comment:
first of all `"0.5"`  is a string that goes against your next condition. Maybe, we can use something like `int(invalid_values) <1` here?

Reply:
thanks @oshtaier for your comments.
```int(invalid_values) <1``` seems to be more appropriate to use here.